torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -20,6 +20,7 @@ std::vector<std::string> _static_quantizable_call_funcs = {
     "linear",
     "batch_norm",
     "hardswish",
+    "elu",
```

Comment:
does it work if you don't add this? I'm trying to remove this list at some point

Reply:
unfortunately tests do fail if we remove this